### PR TITLE
Add docs for release v1.3

### PR DESCRIPTION
**Pull request by addozhang**
- egress gateway

### CODE REVIEW

File: content/en/docs/demos/egress_gateway_passthrough.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+1. Deploy egress gateway via fsm.
+    
+    ```bash
+    helm repo add fsm https://flomesh-io.github.io/fsm
+    helm repo update
+    helm install --namespace fsm --create-namespace --set fsm.version=0.2.0 --set fsm.egressGateway.enabled=true fsm fsm/fsm
+    ```
+
+2. Declear egress gateway.
```
Comment (naqvis) [inline content/en/docs/demos/egress_gateway_passthrough.md]: typo: It should be `Declare egress gateway`

File: content/en/docs/demos/egress_gateway_policy.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+## Egress Gateway passthrough demo
+
+1. Deploy egress gateway via fsm.
+    ```bash
+    helm repo add fsm https://flomesh-io.github.io/fsm
+    helm repo update
+    helm install --namespace fsm --create-namespace --set fsm.version=0.2.0 --set fsm.egressGateway.enabled=true fsm fsm/fsm
+    ```
+2. Declear egress gateway.
```
Comment (naqvis) [inline content/en/docs/demos/egress_gateway_policy.md]: typo: *Declear* should be `Declare`

**Review by naqvis — CHANGES_REQUESTED** [review]

File: content/en/docs/demos/egress_gateway_passthrough.md (hunk 1 above)
Comment (addozhang) [inline content/en/docs/demos/egress_gateway_passthrough.md]:
```suggestion
2. Declare egress gateway.
```

File: content/en/docs/demos/egress_gateway_policy.md (hunk 2 above)
Comment (addozhang) [inline content/en/docs/demos/egress_gateway_policy.md]:
```suggestion
2. Declare egress gateway.
```

**Pull request merged**